clang/test/CXX/drs/dr23xx.cpp

```diff
@@ -252,4 +252,14 @@ namespace dr2397 { // dr2397: 17
     auto (*c)[5] = &a;
   }
 } // namespace dr2397
+
+// CWG2363 was closed as NAD, but its resolution does affirm that
+// a friend declaration cannot have an opaque-enumm-specifier.
+namespace dr2363 { // dr2363: yes
+struct A {
+  friend enum class E; // since-cxx11-error {{reference to enumeration must use 'enum' not 'enum class'}}
```

Comment:
Can you follow the `// expected-error@-1` style of expected directives, like the rest of the file?

Reply:
@Endilll like so?
```cpp
struct A {
  friend enum class E;
  // since-cxx11-error@-1 {{reference to enumeration must use 'enum' not 'enum class'}}
  // expected-error@-2 {{elaborated enumeration type cannot be a friend}}
};
```

Reply:
Yes